tests/ui/target_modifiers/no_value_bool.error_explicit.stderr

```diff
@@ -0,0 +1,13 @@
+error: mixing `-Zreg-struct-return` will cause an ABI mismatch in crate `no_value_bool`
+  --> $DIR/no_value_bool.rs:16:1
+   |
+LL | #![feature(no_core)]
+   | ^
+   |
+   = help: the `-Zreg-struct-return` flag modifies the ABI so Rust crates compiled with different values of this flag cannot be used together safely
+   = note: `-Zreg-struct-return unset` in this crate is incompatible with `-Zreg-struct-return=true` in dependency `enabled_reg_struct_return`
```

Comment:
This doesn't seem correct (unset in the backticks?), plus the string is untranslatable. I'd suggest adding a variant to the strings that explicitly says "unset" and use that

Reply:
Added messages for cases when current crate has unset flag and when external crate has unset flag.